src/libstd/sys/windows/c.rs

```diff
@@ -300,11 +303,12 @@ pub const IMAGE_FILE_MACHINE_ARM64: DWORD = 0xAA64;
 pub const IMAGE_FILE_MACHINE_ARMNT: DWORD = 0x01c4;
 
 #[cfg(not(target_os = "uwp"))]
-pub const EXCEPTION_CONTINUE_SEARCH: LONG = 0;
-#[cfg(not(target_os = "uwp"))]
-pub const EXCEPTION_STACK_OVERFLOW: DWORD = 0xc00000fd;
-#[cfg(not(target_os = "uwp"))]
-pub const EXCEPTION_MAXIMUM_PARAMETERS: usize = 15;
+ifdef! {
```

Comment:
Could the multiple `ifdef!` blocks for uwp be grouped together to have as few `ifdef!` blocks as possible? (should be fine to reorganize the contents here)